### PR TITLE
Changes to CCC Blog Post 

### PR DESCRIPTION
**Pull request by jake-Skybet**
This version should have everything fixed.

### CODE REVIEW

File: _posts/2016-01-11-33C3.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+we instead should take the same approach as medical science in which new products get peer reviewed and scrutinized in the hope of increasing the effectiveness of that drug or product.
+
+The full archive of the event can be found ![here](https://media.ccc.de/b/congress/2016).
+
+![art](/images/ART_INSTAL.jpg)
+
+No matter what time of day or night there was always something going on wither that's a group of friends retro fitting a motor to one of the conference halls chairs and riding it around in the early hours or numerous gatherings about poetry, robotics and drones i lost count of all the different and unique projects people had laid out across their tables in the big assembley halls.
+
+![art installation](/images/ART_INSTAL 2.jpg)
```
Comment (FirsRdJnr) [inline _posts/2016-01-11-33C3.md]: image file name shouldn't contain spaces

File: _posts/2016-01-11-33C3.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+title:      33C3 - Chaos Computer Congress 2016
+author:     jake_roberts
+date:       2016-01-11 08:31:00
+summary:    my experince at 33c3, things i learnt.
+category:   Security
+tags:       security, hacking, 33c3, germany, worksforme, 
+---
+Hallo!
+
+![CCH Hamburg](/images/CCH_OUTSIDE 2.jpg)
```
Comment (FirsRdJnr) [inline _posts/2016-01-11-33C3.md]: image file name shouldn't contain spaces

File: _posts/2016-01-11-33C3.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+layout:     post
+title:      33C3 - Chaos Computer Congress 2016
+author:     jake_roberts
+date:       2016-01-11 08:31:00
```
Comment (FirsRdJnr) [inline _posts/2016-01-11-33C3.md]: Is this an old post that is just being published? If so, does the date want updating to current so it appears as new content or leaving as is?

File: _posts/2016-01-11-33C3.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+![CCH Hamburg](/images/CCH_OUTSIDE 2.jpg)
+I was lucky enough to attend the 33rd Chaos Computer Congress (33c3) last year.
+
+Every year has a motto for example, last years motto was "Gated Communities" this year however, "works for me" was the chosen phrase.
+
+A total of 12k people attended last year and talks were given in three different halls, saal 1, saal 2 and saal G (a smaller room).
+
+(saal 1 and saal 2 could hold around three thousand people)
+
+## A few talks i'd like to share:
```
Comment (tmhn) [inline _posts/2016-01-11-33C3.md]: Isn't it "I'd"? I may be wrong

**Pull request merged**